src/CodeFormatter.cs

```diff
@@ -160,7 +160,9 @@ private static async Task<(Solution solution, int filesFormatted)> FormatFilesIn
 
                 if (filesToFormat != null)
                 {
-                    var fileInArgumentList = filesToFormat.Any(path => document.FilePath.EndsWith(path, StringComparison.OrdinalIgnoreCase));
+                    var fileInArgumentList = filesToFormat
+                        .Select(path => Path.GetRelativePath(Environment.CurrentDirectory, path))

```

Comment:
Calling GetRelativePath here would have it fire over and over again for the same files for each document in the project. It would be better to move the GetRelativePath call to where we split the commandline argument so that we only run it once for each file.

Reply:
Good point! Fixed it